### PR TITLE
convert default arguments to kwargs

### PR DESCRIPTION
**Pull request by ar-india**
Signed-off-by: ar-india <alokranjan2005@gmail.com>

these arguments are not mandatory and hence when not provided in playbook. they are always passed as 'null' to the ansible modules. As a result the default arguments if not provided were always having value as null. This can create problem. 
Hence, have replaced this as kwargs. and then we pass this kwargs through an api remove_null_args(). which removes the param which has value null. This is exactly what we need to do .

### CODE REVIEW

File: ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_group.py [hunk 1]
```diff
@@ -588,7 +588,8 @@ def halt_group(
         group_resp = client_obj.groups.get(id=None, name=group_name)
         if utils.is_null_or_empty(group_resp):
             return (False, False, f"Group '{group_name}' cannot be halted as it is not present.", {})
-        client_obj.groups.halt(id=group_resp.attrs.get("id"), force=force)
+            params = utils.remove_null_args(**kwargs)
```
Comment (rgcostea) [inline ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_group.py]: nit: extra indent

**Review by rgcostea — APPROVED** [review]
lgtm

File: ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_group.py (hunk 1 above)
Comment (ar-india) [reply ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_group.py]: ok

File: ansible_collection/hpe/nimble/plugins/doc_fragments/hpe_nimble.py [hunk 2]
```diff
@@ -38,9 +38,9 @@ class ModuleDocFragment(object):
     type: str
 requirements:
   - Ansible 2.9 or later
-  - Nimble OS 5.0 or later
+  - NimbleOS 5.0 or later
   - HPE Nimble Storage SDK for Python 1.0.0 or later (nimble-sdk Python module)
```
Comment (suneeth51) [inline ansible_collection/hpe/nimble/plugins/doc_fragments/hpe_nimble.py:42]: we may need to request `Python 1.1.0` ?

Comment (ar-india) [reply ansible_collection/hpe/nimble/plugins/doc_fragments/hpe_nimble.py:42]: i don't think so.. our ansible modules can also work with 1.0 as we do not make a call to that api.(forgot the name :( . for which we made the change.. so we are good

**Pull request merged**